static/js/message_edit.js

```diff
@@ -620,6 +620,7 @@ export function save_inline_topic_edit(row) {
     if (!topic_changed) {
         // this means the inline_topic_edit was opened and submitted without
         // changing anything, therefore, we should just close the inline topic edit.
+        $(row).find(".always_visible_topic_edit").show();
         end_inline_topic_edit(row);
```

Comment:
I suspect that the right fix actually involves doing this change inside `end_inline_topic_edit` -- there's other code paths that might call that function, and probably all of them should consider showing/hiding this.  The same may apply to the start code as well.

Reply:
I merged a092e96bffee0caee40cbaa4a4eda6a89b135592 to make the surrounding code more readable; you'll have a merge conflict.

Reply:
Done. I feel `show_edit_topic_on_recipient_row()` and `hide_edit_topic_on_recipient_row()` are better place to hide and show the `always_visible_topic_edit`.